legal/legal/LICENSE.Spring.txt

```diff
@@ -202,9 +202,9 @@
 
 =======================================================================
 
-SPRING FRAMEWORK 4.3.18.RELEASE SUBCOMPONENTS:
+SPRING FRAMEWORK 5.1.8.RELEASE SUBCOMPONENTS:
```

Comment:
did you just change the version in the existing file or did you replace the existing file with the corresponding file from the 5.1.8 artifacts?

Reply:
I have modified the existing file. In case of license file, I will replace it with the file from 5.1.8 artifact.